client/components/site-title/README.md

```diff
@@ -16,4 +16,5 @@ For an example of how to use, have a look at [docs/example.jsx](docs/example.jsx
 - `autoFocusBlogname` - *optional* (bool) Whether to auto-focus the site title input field (defaults to `false`).
 - `blogname` - *optional* (string) Site title to be displayed in corresponding input field.
 - `blogdescription` - *optional* (string) Site tagline to be displayed in corresponding input field.
+- `disabled` - *optional* (bool) Whether title input fields should be disabled (defaults to `false`).
```

Comment:
Maybe we should alter it to `title and description input fields` or `the input fields` because we're disabling both fields when the `disabled` prop is `true`?

Reply:
Yep, sloppiness on my part 👍